OpenScienceJournal/whistlepunk_library/src/main/java/com/google/android/apps/forscience/whistlepunk/metadata/MetaDataManager.java

```diff
@@ -90,6 +90,11 @@ String addOrGetExternalSensor(ExternalSensorSpec sensor,
      */
     void removeSensorFromExperiment(String databaseTag, String experimentId);
 
+    /**
+     * Completely erase references to a sensor from an experiment.
+     */
+    void eraseSensorFromExperiment(String databaseTag, String experimentId);
```

Comment:
There is another implementation of this itnterface that needs to be updated.
OpenScienceJournal/whistlepunk_library/src/sharedTest/java/com/google/android/apps/forscience/whistlepunk/sensordb/MemoryMetadataManager.java

Reply:
Fixed